### ISSUE TITLE
radio and checkboxes structure not correct

### ISSUE THREAD

**Issue by AndreasReichl** [jsonform/jsonform#396]
The Javascript templates for radio and checkboxes is not correct. It should be like:

`<div class="radio"><input type="radio" id="unic_id"><label for="unic_id">Labelname</label></div>`
`<div class="checkbox"><input type="checkbox" id="unic_id"><label for="unic_id">Labelname</label></div>`

Link to W3C: [https://www.w3schools.com/tags/att_input_type_checkbox.asp](url)

**Comment by tchapi** [comment]
Hi @AndreasReichl 

You are right, but then we break the Bootstrap3 element structure (which is `<input>` inside `<label>` as far as I understand (see [here](https://getbootstrap.com/docs/3.4/javascript/#buttons-checkbox-radio)).

Which means: we should update to Bootstrap4 (https://getbootstrap.com/docs/4.6/components/forms/#checkboxes-and-radios)

What do you think ?

**Comment by Lord-EXE** [comment]
Hi tchapi,

Bootrap 4 has been around since 2015 (see [here](https://en.wikipedia.org/wiki/Bootstrap_(front-end_framework))), so you can let version 3 die. I've already rebuilt the whole thing and can send you a solution.

**Comment by tchapi** [comment]
Yes I know, I haven't got much time to update the templates unfortunately :/

How about switching to Bootstrap 5 directly ? I will trigger a major version change anyway ... so that might be worth looking into, no ?

**Comment by AlphaNumeric99** [comment]
@tchapi Any plans for Bootstrap 5 in near future?

**Comment by AndreasReichl** [comment]
Hi Tchapi,

we don´t use Bootstrap anymore. 


Viele Grüße
Andreas Reichl





ANDREAS REICHL
Geschäftsführung.
Mediendesigner für Digital- & Printmedien

Str.			Schulstrasse 2
PLZ / Ort.	92690 Pressath
Tel. 			+49 96 44 / 68 98 13-0
Fax. 		+49 96 44 / 68 98 13-600
Email. 		***@***.*** ***@***.***>
Web. 		www.webprojaggt.de <http://www.webprojaggt.de/>

Ust-Id Nr. DE262919342
Amtsgericht. Weiden i.d. OPf., HRA 2273

Komplementärin: 

WEBPROJAGGT | Verwaltungs GmbH
Amtsgericht. Weiden i.d. OPf., HRB 3419
Geschäftsführung. Andreas Reichl, Sebastian Heinl

Diese E-Mail enthält vertrauliche und/oder rechtlich geschützte Informationen. Wenn Sie nicht der richtige Adressat sind und diese E-Mail irrtümlich erhalten haben, informieren Sie bitte sofort den Absender und vernichten Sie diese E-Mail. Das unerlaubte Kopieren und die unbefugte Weitergabe dieser E-Mail sind nicht gestattet. Der Versand und Empfang von E-Mails dient der Beschleunigung der Kommunikation und der Vorbereitung von Geschäftsabschlüssen sowie der Unterstützung der Kundenbetreuung. Auf dem Wege vom Absender zum Empfänger können E-Mails mitgelesen, verfälscht, nicht oder verspätet zugestellt werden. Aus diesen Gründen können Verträge und Willenserklärungen per E-Mail nicht wirksam abgeschlossen bzw. gegenüber Andreas Reichl abgegeben werden. Diese Erklärungen werden erst nach schriftlicher Bestätigung wirksam.
